src/app/detail/ui/Sidebar.tsx

```diff
@@ -79,7 +81,15 @@ export const Sidebar = ({
     }
   }, [tasks, workflowStates])
 
-  const matches = useMediaQuery('(max-width:600px)')
+  const windowWidth = useWindowWidth()
+  const isMobile = windowWidth < 600 && windowWidth !== 0
```

Comment:
is !== 0 necessary though? I don't see windowWidth ever being 0 if you are taking it from browser.

Reply:
`useWindowWidth` returns `0` initially that's why it was necessary to compare this.